sno/dataset2.py

```diff
@@ -593,3 +618,18 @@ def import_iter_feature_blobs(self, resultset, source):
         for feature in resultset:
             feature_path, feature_content = self.encode_feature(feature, schema)
             yield self.repo_path(feature_path), feature_content
+
+    @property
+    def primary_key(self):
+        # TODO - datasets v2 model supports more than one primary key.
+        # This function needs to be changed when we have a working copy v2 that does too.
+        for column in self.schema:
+            if column.pk_index == 0:
```

Comment:
I think this should raise a NotImplementedError if there are >1, rather than silently returning the first one

Reply:
Good point. Done